workflows_core/dataset/field.py

```diff
@@ -172,29 +172,70 @@ def get_centroids(self):
             "`insert_centroids` not available for non vector_fields"
         )
 
-    def get_keyphrase(self, keyphrase_id: str):
-        raise NotImplementedError(
-            "`get_keyphrase` not available for non keyphrase_fields"
+    def get_keyphrase(self, keyphrase_id: str, alias: str):
+        return self._dataset._api._get_keyphrase(
```

Comment:
shouldnt this only be on the `KeyphraseField` class?

Reply:
Good point - we can double-down on a single entrypoint